src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -225,4 +225,12 @@ public void testPullRequestAnalysisWithNewBlockerAndCriticalIssues() {
 
     verify(pullRequestFacade).createOrUpdateSonarQubeStatus(GHCommitState.ERROR, "SonarQube reported 2 issues, with 1 critical and 1 blocker");
   }
+
+  @Test
+  public void should_update_sonarqube_status_even_if_unexpected_errors_were_raised() {
+    // raises an NPE due to improper mock setup; not realistic, but good enough for this test
+    pullRequestIssuePostJob.execute(context);
+
+    verify(pullRequestFacade).createOrUpdateSonarQubeStatus(GHCommitState.ERROR, "unexpected error occurred while posting comments");
```

Comment:
Missed that it's the details message.. 

> Just to make sure - we're talking about this message, right?
> ![image](https://cloud.githubusercontent.com/assets/8449713/22067046/446edea6-dd98-11e6-949c-b7a5307f5411.png)
> (The highlighted part)

So I'd suggest to **_comment on the PR_** with the (hidden) details and the message. That's something that IMO needs attention by all PR subscribers: _An essential check has failed_.
Also, that would benefit much from the error context .